test/unit/player-play.test.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-env qunit */
+import sinon from 'sinon';
+import {silencePromise} from '../../src/js/utils/promise';
+import TestHelpers from './test-helpers';
+
+QUnit.module('Player#play', {
+
+  beforeEach() {
+    this.clock = sinon.useFakeTimers();
+    this.player = TestHelpers.makePlayer({techOrder: ['html5']});
```

Comment:
if the html5 tech is required for testing, we'd want to wrap this file in a `!IS_IE8`.

Reply:
I don't think it actually is. It can probably use the fake tech.